uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/absence-tab.js

```diff
@@ -9,14 +9,19 @@
     }
   });
 
-  require([
-    'leave-absences/shared/config',
-    'leave-absences/absence-tab/app'
-  ], function () {
-    document.dispatchEvent(typeof window.CustomEvent == "function" ? new CustomEvent('absenceTabReady') : (function () {
-      var e = document.createEvent('Event');
-      e.initEvent('absenceTabReady', true, true);
-      return e;
-    })());
+  require(['leave-absences/shared/config'], function(__){
+    require(
+      [
+        'leave-absences/absence-tab/app'
```

Comment:
@AkA84 Well, app.js asks for ui-router as a dep, and config.js redefines the path to the ui-router module. Since define works async as well, if we do:

![image](https://cloud.githubusercontent.com/assets/3973243/26826671/cc967d1c-4ab1-11e7-930b-a20c137552c8.png)

Just tested and it now fails 100% of time, which is expected.